src/scripts/scripts/zone/caverns_of_time/old_hillsbrad/boss_captain_skarloc.cpp

```diff
@@ -284,8 +287,8 @@ struct boss_captain_skarlocAI : public ScriptedAI
         else
             DevotionAura_Timer -= diff;
 
-        if(HeroicMode)
-        if(Consecration_Timer < diff)
+        if (HeroicMode)
```

Comment:
This if can be removed when setting Consecration_Timer in reset. I would use Consecration_Timer = HeroicMode ? 8000 : 0;